EventFilter/L1TRawToDigi/plugins/implementations_stage2/EtSumUnpacker.cc

```diff
@@ -91,6 +91,15 @@ namespace stage2 {
 
        res_->push_back(bx,ht);
 
+
+       //HI-SUM
+
+       l1t::EtSum towCount = l1t::EtSum();
```

Comment:
@rekovic i guess we can have "raw" data around without this new piece of information? Or is that not possible? If so, some protection is needed when the new information is not present.